apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/CommonDataTableFixture.cs

```diff
@@ -26,21 +26,20 @@ public class CommonDataTableFixture : SpannerTableFixture
     {
         public int RowCount { get; } = 15;
 
-        private bool _createIndexes;
-
-        public CommonDataTableFixture(string tableName, bool createIndexes) : base(tableName)
+        public CommonDataTableFixture(string tableName) : base(tableName)
         {
-            _createIndexes = createIndexes;
```

Comment:
This never worked, as the `CreateTable` method is called from the base constructor. This meant that `CreateTable` was executed before `_createIndexes` was set.